minidump-processor/Cargo.toml

```diff
@@ -38,3 +38,4 @@ scroll = "0.10.2"
 
 [dev-dependencies]
 test-assembler = "0.1.5"
+synth-minidump = { path = "../synth-minidump" }
```

Comment:
This is a good idea and I'm not sure why I never got around to doing it!

Reply:
I assumed you (reasonably) thought the subcrates could grab it as a pub symbol in test builds but sadly It Doesn't Work Like That/